README.md

```diff
@@ -1,12 +1,14 @@
 # Gadi-benchmarking
-Template scripts to automate submission of identical benchmark tasks with increasing compute resources
+Template scripts to automate submission of identical benchmark tasks with increasing compute resources. 
+
+This is to benchmark resources for multi-threading tools, to help determine the optimal number of CPUs and queue type for the job on a single task before submtting the job on many tasks. Benchmarking in this way is critical to ensuring efficient and responsible use of HPC, saving time and compute costs in the long run. 
 
 ## Overview
 Given the need to frequently benchmark bioinformatics tools when developing pipelines, this repository contains a pair of sample scripts that can be adapted for use to other tools. 
 
 The template consists of a pair of scripts:
-* `<tool>_benchmark_run.sh` sets up the resources for each benchmark run
-* `<tool>_benchmark.pbs` is launched by the above, once for each identical analysis at the different resource thresholds
+* `<tool>_benchmark_run.sh` : sets up the resources for each benchmark run
```

Comment:
Don't think this is needed.

Reply:
Fixed - [d99f7d3](https://github.com/Sydney-Informatics-Hub/Gadi-benchmarking/pull/1/commits/d99f7d3c75a55c261386ecaf41b8f79116c884cf) and [9dbbf76](https://github.com/Sydney-Informatics-Hub/Gadi-benchmarking/pull/1/commits/9dbbf760048d1be3bcf7959cc85e6c731d8a0c89)